src/components/Jobs/RunningJobs/JobDetails/JobDetails.js

```diff
@@ -42,15 +42,17 @@ const JobDetails = props => {
         <span data-test-running-job-file-name>{get(exportedFiles, '0.fileName')}</span>
       </div>
       <div className={css.delimiter}>
-        <span data-test-running-job-hr-id>{get(job, 'hrId', '')}</span>
-        {runBy && (
-          <span data-test-running-job-triggered-by>
-            <FormattedMessage
-              id="ui-data-export.triggeredBy"
-              values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
-            />
-          </span>
-        )}
+        <span data-test-running-job-hr-id>{hrId}</span>
+        <span data-test-running-job-triggered-by>
+          {runBy && (
+            <span data-test-running-job-triggered-by>
+              <FormattedMessage
+                id="ui-data-export.triggeredBy"
+                values={{ userName: `${runBy.firstName} ${runBy.lastName}` }}
+              />
+            </span>
+          )}
+        </span>
```

Comment:
@VictorSoroka1 
That changes was before my last commit, so I decided just revert my changes and leave this ones